common/remote/sockfile.cpp

```diff
@@ -4283,7 +4283,6 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
         file->rename(toname);
         reply.append((unsigned)RFEnoerror);
         return true;
-        return false;
```

Comment:
It does matter..

Any remote rename whose result was checked would have seen a false negative.